automation/src/test/scala/org/broadinstitute/dsde/workbench/pipeline/PipelineInjector.scala

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.dsde.workbench.pipeline
```

Comment:
Is there still planned work to make this a shared lib code, or is it sufficiently different from the rawls code to warrant the duplication?

Reply:
At this time, the shared lib discussion is still ongoing. It looks more likely teams are adopting the new pipeline into their e2e tests tailored to their specific language. So stay tuned and I'll circle back if anything changes.